bin/mocha

```diff
@@ -65,7 +69,14 @@ process.argv.slice(2).forEach(function (arg) {
   }
 });
 
-var proc = spawn(process.execPath, args, { stdio: 'inherit' });
+var binary = process.execPath;
+if (gdb) {
+  args.unshift(binary);
+  args.unshift('--args');
+  binary = '/usr/bin/gdb';
```

Comment:
Is GDB guaranteed to be installed at `/usr/bin`? In every distro? When running on Mac via MacPorts or Homebrew? When running on Windows via MinGW or Cygwin? As a rule, it would probably be safer to assume that it's on the path and just use the executable name, I think. (This is a moot point if we don't add code specifically for GDB, but I want to throw it out there for reference.)